BubbleSort/BubbleSort/BubbleSort.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace BubbleSort;
+
+/// <summary>
+/// Class that has method of sorting list using bubble sort algorithm.
+/// </summary>
+public class BubbleSorter
```

Comment:
```suggestion
public static class BubbleSorter
```
В нём только static-методы и вообще нет полей, нет причин быть не static